docs/misc/faq.md

```diff
@@ -101,7 +101,7 @@ with several powerful functionalities:
  where thousands of people have uploaded useful containers: anything from Redis,
  CouchDB, PostgreSQL to IRC bouncers to Rails app servers to Hadoop to base 
  images for various Linux distros. The
- [*registry*](../reference/api/registry_index_spec/#registryindexspec) also
+ [*registry*](/reference/api/hub_registry_spec/) also
```

Comment:
I think we should change this to `/registry/`. The current link leads to the "Registry V1 specs", which may be removed soon.

Note that you will get a 404-error if you test the `/registry/` link with `make docs`, that's because that page is in another repository (https://github.com/docker/distribution/blob/master/docs/index.md). It's safe to ignore that 404-error, because the page *does* exist; https://docs.docker.com/registry/